src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int RANDOM_BOUND = Color.values().length;
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(RANDOM_BOUND);
+        return Color.values()[index].name().toLowerCase();
```

Comment:
The use of toLowerCase() changes the case of the enum constant's name. According to the checklist, you should use the name() method without changing the case of the enum constant. This is because enum names are usually constants and should be returned as they are declared.